src/main/java/calculator/repository/MemorySeparatorRepository.java

```diff
@@ -0,0 +1,19 @@
+package calculator.repository;
+
+import calculator.model.Separator;
+import java.util.HashSet;
+import java.util.Set;
+
+public class MemorySeparatorRepository implements SeparatorRepository{
```

Comment:
```suggestion
public class InMemorySeparatorRepository implements SeparatorRepository{
```
사소하지만 InMemory가 더 이름이 적절해보입니다!